kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/OpenSearchRequest.java

```diff
@@ -85,14 +85,20 @@ private static String getQueryString(JsonNode body) {
     return queryString;
   }
 
+  // This is required. consider returning a more useful error to the user than 500
+  // or setting a reasonable default (default seems a good option here)
   private static int getHowMany(JsonNode body) {
     return body.get("size").asInt();
   }
 
+  // This is required. consider returning a more useful error to the user than 500
+  // or setting a reasonable default (not sure what that would be)
```

Comment:
probably should return something like this instead of 500
{
  "error": {
    "code": 400,
    "message": "Bad Request",
    "details": "The 'query.gte' field is required but missing in the request."
  }
}